pkg/mfa/client.go

```diff
@@ -263,7 +270,7 @@ func (c *Client) VerifyFactor(
 	})
 	responseBody := bytes.NewBuffer(postBody)
 
-	endpoint := fmt.Sprintf("%s/auth/factors/verify", c.Endpoint)
+	endpoint := fmt.Sprintf("%s/auth/challenges/%s/verify", c.Endpoint, opts.AuthenticationChallengeID)
```

Comment:
Could we also remove the `authentication_challenge_id` field from the `postBody` above?

Reply:
Done, sorry for the delay